LICENSE.md

```diff
@@ -1,4 +1,6 @@
-Copyright (c) 2013-2014 Eduardo Gurgel Pinho
+# The MIT License
+
+Copyright (c) 2013 Eduardo Gurgel <eduardo@gurgel.me>
```

Comment:
I'm not sure about changing the copyright line?

Reply:
@nathany-copia IANAL. If I remember correctly, for most cases, using a single year (the year of initial creation) is sufficient and the recommended approach as we don't have to constantly update the latest copyright year.